src/core/public/chrome/ui/header/__snapshots__/header.test.tsx.snap

```diff
@@ -2584,12 +2698,12 @@ exports[`Header handles visibility and lock changes 1`] = `
                       onClick={[Function]}
                     >
                       <img
-                        alt="OpenSearch Dashboards logo"
+                        alt="opensearch dashboards logo"
```

Comment:
`applicationTitle` is still handled by individual components using `branding` and each have their own defaults; some use `OpenSearch Dashboards` and some use `opensearch dashboards`. The upcoming branding service will normalize that.